pkg/op-brains/op_brains/documents/__init__.py

```diff
@@ -9,9 +9,9 @@
 import time
 
 chat_sources = [
-    [
-        FragmentsProcessingStrategy,
-    ],
+    # [
+    #     FragmentsProcessingStrategy,
+    # ],
```

Comment:
how come? 🤔

Reply:
I sent a message on slack about this https://bleu-builders.slack.com/archives/C0740EA570D/p1725553560473179